chromium_src/third_party/devtools-frontend/src/scripts/build/typescript/ts_library.py

```diff
@@ -3,51 +3,49 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this file,
 # You can obtain one at http://mozilla.org/MPL/2.0/.
 
-import shutil
 import argparse
-import atexit
 import os
 import re
 
 import brave_chromium_utils
 import override_utils
 
 
-def remove_copied_sources(sources):
-    for source in sources:
-        if os.path.exists(source):
-            os.remove(source)
+def ensure_hardlink(src, dst):
```

Comment:
why are we using hard links here? Also isn't `link` below the async version? Same with unlink? Looks like they are being treated as if they are sync

Reply:
cc: @boocmp

Reply:
Because on windows the sym links requires an admin rights. They are sync as I know.

Reply:
I guess also I'm wondering why we changed this in the first place to use links

Reply:
oic, this is `os.link` instead of `fs.link` and no callback so it is sync

Reply:
this isn't a blocker or anything, I'm just curious about the change